wagtail/models/__init__.py

```diff
@@ -783,6 +783,127 @@ def get_preview_template(self, request, mode_name):
         )
 
 
+class WorkflowMixin:
+    @classmethod
+    def get_default_workflow(cls):
```

Comment:
Not sure... It won't be called anywhere on a Page by wagtail, but I don't see why we should stop developers (if they so wish) to make use of this method to have a default workflow based on the specific Page model class rather than the `WorkflowPage` link. They'll probably need to change some other stuff as well, but I'd assume they'll know what they're doing anyway.